pkg/packagekit/updates.jsx

```diff
@@ -500,6 +536,8 @@ class OsUpdates extends React.Component {
 
                 // when GetUpdates() finished, get the details for all packages
                 Finished: () => {
+                    // for interactively testing layout changes with large updates; never commit this!
+                    //injectMockUpdates(updates);
```

Comment:
This is hidden pretty deeply in the code... could we use a combination of `injectMockUpdates !== undefined` and another global variable such as `const injectLargeUpdates = true`?